src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,37 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { filterSlice } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const status = useAppSelector(state => state.filter.status);
+  const query = useAppSelector(state => state.filter.query);
+
+  const statusChange = (newStatus: Status) =>
+    dispatch(filterSlice.actions.setStatus(newStatus));
+
+  const queryChange = (newQuery: string) =>
+    dispatch(filterSlice.actions.setQuery(newQuery));
```

Comment:
lets name event handlers according to common [naming convention](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)